pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.en-gb.md

```diff
@@ -0,0 +1,181 @@
+
+### 9.2.4 SecNumCloud Connectivity: SPN and VPN
+
+An optional connectivity service is available on SecNumCloud qualified infrastructure. This service option includes:
+
+- A Secure Private Network  (SNC Secured Private Network or SPN);
+- A secure interconnection between remote SPNs (SNC SPN Inter-DC);
+- An IPsec VPN gateway (SNC VPN Gateway).
+
+An overview of these options is available on [this link](/products/hosted-private-cloud-hosted-private-cloud-powered-by-vmware-networking-secnumcloud-connectivity).
+
+### 9.2.5 Data encryption
+
+On a SecNumcloud or non-SecNumcloud qualified infrastructure, you can apply encryption at rest using the vNKP brick available on the service to encrypt VMs or at the datastore level of a vSAN Cluster.
+
+The same operation can be carried out if the customer chooses to use a KMS external to the OVHcloud solution.
+
+Instructions for use are available on [this link](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vm_encrypt-vnkp).
+
+### 9.2.6 Advanced security with HDS
+
+You can benefit from an advanced security pack by enabling HDS options on your infrastructure.
+
+The pack includes several features such as: [token validator](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/interface-secure), access via 2FA, session timeout, fail2ban, hids, force TLS v1.2 protocol, etc.
+
+## 10 - Reversibility
+
+To ensure data portability and reversibility on the service, OVHcloud allows you to import and export your data (virtual machines and vCenter configuration files) autonomously in a VMDK file format or any other format supported by the VMware hypervisor. You can also use the APIs provided to facilitate these operations.
+
+OVHcloud’s portability principles are described in its own [reversibility policy](/pages/account_and_service_management/reversibility/00-global-reversibility-policy) and those specific to the Hosted Private Cloud by VMware service are set out [specific policy](/pages/account_and_service_management/reversibility/02-hosted-private-cloud-reversibility-policy).
+
+### 10.1 - Erasure of customer data 
+
+Following the customer’s decommission of the service and prior to the removal of the hard drive from the rack, an erasing robot applies a secure data erasure procedure based on the NIST SP 800-88 r1 level ‘Purge’. In case of technical constraints or limitations on certain ranges of hard drives and when the level 'Purge' cannot be applied, the erase at the level 'Clear' will run.
+
+On an infrastructure qualified as SecNumCloud, the 'Destroy' level is applied after the disk has been extracted from the array.
+
+### 10.2 - Erasure of technical data
+
+Following the customer’s resignation from the service, OVHcloud releases the resources allocated to him and deletes the configurations made during service delivery
+
+
```

Comment:
Hello, in the FR version there are 11 points, but the EN version only has 10, is that intentional?

Reply:
this part should not be translated, it should be only in French  (it's required by new version HDS version)

Reply:
alright, thanks!